__device-tests__/utils.js

```diff
@@ -5,19 +5,72 @@ import jimp from 'jimp';
 
 const { isAndroid } = e2eUtils;
 
+/**
+ * Helper to take a screenshot of an element.
+ *
+ * @param {*}      element           Element instance.
+ * @param {Object} options           Options
+ * @param {number} [options.padding] Padding offset to apply to the screenshot.
+ * @return {Buffer} Sreenshot image.
+ */
+export async function takeScreenshotByElement( element, { padding } = {} ) {
+	const sessionCapabilities = await editorPage.driver.sessionCapabilities();
+	const { pixelRatio } = sessionCapabilities;
+
+	const location = await element.getLocation();
+	const size = await element.getSize();
+	const crop = {
+		x: location.x,
+		y: location.y,
+		width: size.width,
+		height: size.height,
+	};
+	// Location and size value are in pixels on iOS, we need to
+	// convert them to device coordinates.
+	if ( ! isAndroid() ) {
+		crop.x *= pixelRatio;
+		crop.y *= pixelRatio;
+		crop.width *= pixelRatio;
+		crop.height *= pixelRatio;
+	}
+	if ( padding ) {
+		// The padding value is in pixels, so we need to convert
+		// it to device coordinates.
+		padding *= pixelRatio;
+
+		crop.x -= padding;
+		crop.y -= padding;
+		crop.width += padding * 2;
+		crop.height += padding * 2;
+	}
+	return takeScreenshot( { crop } );
+}
+
 /**
  * Helper to take a screenshot and manipulate it.
  *
  * @param {Object}  options                    Options
  * @param {boolean} [options.withoutKeyboard]  Prevents showing the keyboard in the screenshot.
  * @param {number}  [options.heightPercentage] Specify a custom height in percentage.
+ * @param {Object}  [options.crop]             Specify values to crop the screenshot.
+ * @param {number}  [options.offset.x]         X offset to crop.
+ * @param {number}  [options.offset.y]         Y offset to crop.
+ * @param {number}  [options.offset.width]     Width offset to crop.
+ * @param {number}  [options.offset.height]    Height offset to crop.
```

Comment:
I do not see any references to `offset`, should this be `crop` instead?

```suggestion
 * @param {number}  [options.crop.x]         X offset to crop.
 * @param {number}  [options.crop.y]         Y offset to crop.
 * @param {number}  [options.crop.width]     Width offset to crop.
 * @param {number}  [options.crop.height]    Height offset to crop.
```

Reply:
Yep, I originally named the option `offset` and then changed it to `crop`, but I forgot to update this part. I'll update it.

Reply:
I fixed this in https://github.com/wordpress-mobile/gutenberg-mobile/pull/6038/commits/a9885842f0899fb4d64a03bff735486f08b1786a, although I changed a bit the structure in https://github.com/wordpress-mobile/gutenberg-mobile/pull/6038/commits/c718739bfed8508e39b0c168d968894928741fdc